fe/src/main/java/org/apache/doris/plugin/PluginContext.java

```diff
@@ -18,4 +18,14 @@
 package org.apache.doris.plugin;
 
 public class PluginContext {
+
+    private String pluginJarPath;
+
+    public void setPluginJarPath(String pluginJarPath) {
```

Comment:
```suggestion
    protected void setPluginJarPath(String pluginJarPath) {
```
Let user use the method is not a good idea

Reply:
OK